specs/mediawiki/v1/content.yaml

```diff
@@ -220,11 +220,13 @@ paths:
       tags:
         - Page content
       description: >
-        Retrieve the latest html for a page title.
+        Retrieve the latest html for a page title among known to RESTBase.
```

Comment:
Also, `s/known to RESTBase/available in storage/` (clients shouldn't care it's RESTBase responding to the requests)